### ISSUE TITLE
Examples are broken on master

### ISSUE THREAD

**Issue by arporter** [stfc/PSyclone#228]
Some of the examples fail because they have not been updated with the new way of handling the Configuration object. This is my fault because I reviewed the associated ticket and merged it to master.

I'll take this opportunity to fix the test failures that I see when running in parallel.
I'll also look at running the examples on Travis.

**Comment by arporter** [comment]
The test that fails (on 4 cores on my desktop) is `config_test.test_missing_file`.

**Comment by arporter** [comment]
I was going to add a pytest `setup_module` fixture to fix this but then I remembered that I've already done that in my `write_kernels` branch so I won't do that here.

**Comment by rupertford** [comment]
PR #232 has been merged to master. Closing this issue.
